crates/parser/src/ast.rs

```diff
@@ -43,6 +49,16 @@ pub enum AssertionKind {
         #[serde(with = "optional_timeout")]
         timeout: Option<Duration>,
     },
+    CalcMetrics {
+        node_name: NodeName,
+        metric_name: String,
+        math_ops: MathOps,
+        metric_name_calc: String,
+        op: Operator,
+        target_value: u64,
+        #[serde(with = "optional_timeout")]
+        timeout: Option<Duration>,
+    },
```

Comment:
I think we can call the metrics a/b, something like this

```suggestion
    CalcMetrics {
        node_name: NodeName,
        metric_name_a: String,
        math_ops: MathOps,
        metric_name_b: String,
        op: Operator,
        target_value: u64,
        #[serde(with = "optional_timeout")]
        timeout: Option<Duration>,
    },
```
I think is more clear that `_calc` wdyt?